src/Interpreters/Context.h

```diff
@@ -729,7 +729,11 @@ class Context: public std::enable_shared_from_this<Context>
     BackupsWorker & getBackupsWorker() const;
 
     /// I/O formats.
-    InputFormatPtr getInputFormat(const String & name, ReadBuffer & buf, const Block & sample, UInt64 max_block_size, const std::optional<FormatSettings> & format_settings = std::nullopt) const;
+    /// If number_of_streams is set then the number of threads are divided among the streams
+    /// Eg: max_threads = 16, number_of_streams = 8, then each stream will use 2 threads
```

Comment:
this comment seems to imply that we use `settings.max_threads / number_of_streams` instead of `settings.max_threads / storage->paths.size()`, so update the comment or update the code?